tqdm/_tqdm.py

```diff
@@ -757,6 +760,9 @@ def set_description(self, desc=None):
     def moveto(self, n):
         self.fp.write(_unicode('\n' * n + _term_move_up() * -n))
 
+    def _is_complete(self):
+        """(Needs refactoring) A way to see if a progress bar and/or task has finished"""
```

Comment:
Why does this need refactoring? `TODO: needs refactoring` at the end may be better.